docs/release-notes/2024-03-19-new-field-isMarketplaceResponsibleForTaxes-checkout-api.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Tax Service"
```

Comment:
Achei esse título pouco explicativo... Talvez algo tipo "New field for tax calculation on Checkout API" seja melhor

Reply:
Obrigada pela sugestão! Foi total falta de atenção aqui, eu acabei ajustando a slug, mas esqueci do title